ludwig/data/prompt.py

```diff
@@ -169,44 +174,64 @@ def format_input_with_prompt(
             template = DEFAULT_ZERO_SHOT_PROMPT_TEMPLATE
```

Comment:
OOC: Would a user be able to use the `DEFAULT_ZERO_SHOT_PROMPT_TEMPLATE` for the prompt template within an input feature definition like so?

```
input_features:
    - name: context
      type: text
      preprocessing:
          prompt:
            template: None  # Use default.
```

Reply:
Yes, though I believe in this case they need to provide a `task`.